docs/settings/settings.md

```diff
@@ -0,0 +1,175 @@
+# Settings
+
+This page contains a list of advanced settings which can be used in Harvester.
+You can modify the custom resource `settings.harvesterhci.io` from the Dashboard UI or with the `kubectl` command.
+
+## `backup-target`
+
+This setting allows you to set a custom backup target to store VM backups. It supports NFS and S3. 
+For further information, please refer to the [Longhorn documentation][longhorn-backup-target]. 
+
+Default: none
+
+[longhorn-backup-target]: https://longhorn.io/docs/1.2.2/snapshots-and-backups/backup-and-restore/set-backup-target/#set-up-aws-s3-backupstore
+
+#### Example
+
+```json
+{
+  "type": "s3",
+  "endpoint": "https://s3.endpoint.svc",
+  "accessKeyId": "test-access-key-id",
+  "secretAccessKey": "test-access-key",
+  "bucketName": "test-bup",
+  "bucketRegion": "us‑east‑2",
+  "cert": "",
+  "virtualHostedStyle": false
+}
+```
+
+## `http-proxy`
+
+This setting allows you to configure an HTTP proxy to access external services, including the download of images and backup to s3 services.
+Default: `{}`
+
+The following options and values can be set:
+
+- Proxy URL for HTTP requests: `"httpProxy": "http://<username>:<pswd>@<ip>:<port>"` 
+- Proxy URL for HTTPS requests: `"httpsProxy": "https://<username>:<pswd>@<ip>:<port>"`
+- Comma-separated list of hostnames and/or CIDRs: `"noProxy": "<hostname | CIDR>"`
+
+#### Example
+
+```json
+{
+  "httpProxy": "http://my.proxy",
+  "httpsProxy": "https://my.proxy",
+  "noProxy": "some.internal.svc"
+}
+```
+
+## `log-level`
+
+This setting allows you to configure the log level for the Harvester server. 
+
+Default: `info`
+
+The following values can be set. The list goes from the least to most verbose log level:
+
+- `panic`
+- `fatal`
+- `error`
+- `warn`, `warning`
+- `info`
+- `debug`
+- `trace`
+
+#### Example
+
+```
+debug
+```
+
+## `overcommit-config`
+
+This setting allows you to configure the percentage for resources overcommit on CPU, memory, and storage. By setting resources overcommit, this will permit to schedule additional virtual machines even if the the physical resources are already fully utilized.
+
+Default: `{ "cpu":1600, "memory":150, "storage":200 }`
+
+The default CPU overcommit with 1600% means, for example, if the CPU resources
+limit of a virtual machine is `1600m` core, Harvester would only request `100m`
+CPU for it from Kubernetes scheduler.
+
+#### Example
+
+```json
+{
+  "cpu": 1000,
+  "memory": 200,
+  "storage": 300
+}
+```
+
+## `server-url`
+
+This setting displays the URL of Harvester server. If you've configured virtual IP (VIP) correctly, the value should be the same as VIP.
```

Comment:
```suggestion
This setting displays the Harvester server URL, used for the Harvester management endpoint and GUI URL. If a [virtual IP (VIP)][harvester-vip] has been set, then the value displayed will be the VIP address instead of the host IP.

[havester-vip]: https://docs.harvesterhci.io/v0.3/install/harvester-configuration/#installvip
```

Reply:
Updated. Also changed absolute links to relative. Thanks